Data/Questions/so1.txt

```diff
@@ -125,17 +125,17 @@ v Se viene usata la lista di blocchi liberi, c'è un overhead di spazio, contrar
 > Se ci sono blocchi da 1kB, e il disco contiene 1TB, l'occupazione dovuta alla lista di blocchi liberi è dell'1%
 > Se viene usata la lista di blocchi liberi, una parte viene memorizzata su disco ed una parte in memoria principale
 
-24) UNSAFE Quale delle seguenti azioni va effettuata sia per un process switch che per un mode switch, assumendo di essere in un SO nel quale le funzioni di sistema sono eseguite all'interno dei processi utente?
+24) UNSAFE (sbagliata secondo Gabriele Pelissetto) Quale delle seguenti azioni va effettuata sia per un process switch che per un mode switch, assumendo di essere in un SO nel quale le funzioni di sistema sono eseguite all'interno dei processi utente?

```

Comment:
Secondo me questa è in realtà giusta perché per esclusione le altre sono false

Reply:
Lascio solo il mark unsafe